client/my-sites/upgrades/checkout/cart-toggle.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { showCartOnMobile } from 'lib/upgrades/actions';
+
+class CartToggle extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { isShowingCartOnMobile: false };
+	}
+
+	toggleCartOnMobile = ( event ) => {
+		event.preventDefault();
+
+		const show = ! this.state.isShowingCartOnMobile;
+		this.setState( { isShowingCartOnMobile: show } );
+		showCartOnMobile( show );
+	};
+
+	render() {
+		const label = this.state.isShowingCartOnMobile
+			? this.props.translate( 'Hide order summary' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Order Summary' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).